code/frameworks/solidjs-vite/README.md

```diff
@@ -0,0 +1,46 @@
+# Storybook for Solid <!-- omit in toc -->
+
+## Requirements
+
+- [Solid](https://www.solidjs.com/) >= 1.x
+- [Storybook](https://storybook.js.org/) >= 7.x
+
+## Getting Started
+
+### In a project without Storybook
```

Comment:
To automatically setup the solidjs-vite framework and to install it automatically for the user, you have to make some adjustments in https://github.com/storybookjs/storybook/blob/next/code/lib/cli/src/project_types.ts#L123 and https://github.com/storybookjs/storybook/blob/next/code/lib/cli/src/initiate.ts#L67

Reply:
@valentinpalkovic  I made changes to the needed files, if there is something else please guide me